authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -761,18 +772,36 @@ public Boolean onGroupAdded(Group group) throws AuthorizationPluginException {
 
   @Override
   public Boolean onGroupRemoved(Group group) throws AuthorizationPluginException {
-    VXGroupList list = rangerClient.searchGroup(ImmutableMap.of("name", group.name()));
-    if (list.getListSize() == 0) {
-      LOG.warn("The group({}) doesn't exist in the Ranger!", group);
+    VXGroupList vxGroupList = rangerClient.searchGroup(ImmutableMap.of("name", group.name()));
+    boolean isExist = false;
+    long groupId = -1;
```

Comment:
I think maybe we can use LONG groupId or Option(long) groupId to replace `long`

Reply:
OK.